test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -7075,8 +7075,353 @@ BOOST_AUTO_TEST_CASE(calling_nonexisting_contract_throws)
 	BOOST_CHECK(callContractFunction("h()") == encodeArgs(u256(7)));
 }
 
+BOOST_AUTO_TEST_CASE(const_fixed_type_return)
+{
+	char const* sourceCode = R"(
+		contract C {
+			fixed public constant a = 3.5;
```

Comment:
Probably the `constant` modifier is not needed in the test. (It might change in the future also.)